runtime/calamari/src/lib.rs

```diff
@@ -371,13 +370,14 @@ impl pallet_balances::Config for Runtime {
 }
 
 parameter_types! {
-    /// Relay Chain `TransactionLengthToFeeCoeff` / 10
-    pub const TransactionLengthToFeeCoeff: Balance = mKMA / 100;
+    /// Relay Chain `TransactionLengthToFeeCoeff` / 10 (1_000_000 on Kusama)
+    pub const TransactionLengthToFeeCoeff: Balance = mKMA;
```

Comment:
This 100x makes almost no difference. Even if we I add the maximum length in the test.